docs/prototypes/form-builder-redesign-prototype.html

```
@@ -0,0 +1,755 @@
+<title>Dculus Forms — Builder Redesign Prototype</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)